packages/engine/README.md

```diff
@@ -103,41 +101,20 @@ Depending on your needs, different dependencies are required. Building this proj
         git checkout $latestTag
         ```
 
-- Python [3.7.x] is required for Python initialization or Python behaviors.
-
+- Python [3.10.x] is required for Python initialization and Python behaviors.
```

Comment:
Is Python 3.10 _required_? We should probably put _some_ effort into finding out the minimum version we support at the moment.

Reply:
I agree but (without checking) I guess, that Python 3.10 is not distributed on all flavors. I guess we can say, that we target Python 3.10 but it *may* also work with older Python versions.

Reply:
Maybe we should add a `Python` field on https://github.com/hashintel/hash/blob/main/.github/ISSUE_TEMPLATE/bug-report-hash-engine.md